.github/workflows/create-v1.0-pull-request.yml

```diff
@@ -14,8 +14,8 @@ on:
     branches:
       - "v1.0/pipelinebuild/*"
     paths:
-      - 'src/**/*.php'
-      - !'src/Core/GraphConstants.php'
+      - 'src/Model/**/*.php'
+      - 'src/CallRecords/Model/**/*.php'
```

Comment:
This will fail if `TermStore` goes GA and we have a diff only under that folder.

Does this accept the following specification?
```
src/**/Model/**/*.php
```

Reply:
Yes, but then that would trigger for beta as well. It is helpful to keep the separate. I'll add a path spec for termstore.

Reply:
Changed my mind. I added an exclude spec.